scholarmarkdown.gemspec

```diff
@@ -78,6 +78,8 @@ Gem::Specification.new do |s|
     s.specification_version = 4
   end
 
+  is_mac = RUBY_PLATFORM.include?('darwin')
```

Comment:
Are the changes to this file done manually, or using the rakefile?
Because this file is regenerated upon every using `rake`.

Reply:
They were manually 🙈, I've now generated the updated gemspec using `rake gemspec`